src/llmq/quorums_signing_shares.cpp

```diff
@@ -1133,8 +1133,22 @@ void CSigSharesManager::Cleanup()
                 assert(m);
 
                 auto& oneSigShare = m->begin()->second;
-                LogPrintf("CSigSharesManager::%s -- signing session timed out. signHash=%s, id=%s, msgHash=%s, sigShareCount=%d\n", __func__,
-                          signHash.ToString(), oneSigShare.id.ToString(), oneSigShare.msgHash.ToString(), count);
+
+                std::string strMissingMembers;
+                if (LogAcceptCategory("llmq")) {
+                    auto quorum = quorumManager->GetQuorum((Consensus::LLMQType)oneSigShare.llmqType, oneSigShare.quorumHash);
```

Comment:
This version of `GetQuorum` temporary locks `cs_main` inside, could lead to dead locks.

Reply:
Nice catch! I pushed 2 commits that lead to this place being lock-less. It became a small refactoring however which is unrelated to what the PR title says...